153_find_minimum.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+
+class Solution:
```

Comment:
The problem says "You must write an algorithm that runs in O(log n) time", but the time complexity of this solution is O(n). So rewrite this code so that the time complexity is O(log n).